Assets/AnkrSDK/Scripts/Data/SupportedWallets.cs

```diff
@@ -0,0 +1,46 @@
+using System.Linq;
+
+namespace AnkrSDK.Data
+{
+	public static class SupportedWallets
```

Comment:
why dont you just create 2 separate enums? I don't see any profit from making it using one enum and custom property drawers.

Reply:
separating the enums will create an easier maintenance and provide more code readability

Reply:
you can then add a helper to transform from one enum to another

Reply:
Then we will have different interfaces for the same feature, isn't it?

Reply:
Why is it?

Reply:
Because we use Wallet enum in IWalletHandler interface. In that case we need to do specific interface for WebGL to avoid this. Or use one Wallet enum for all platform.

<img width="492" alt="image" src="https://user-images.githubusercontent.com/38144180/187137946-75f047a6-f62b-4314-af11-6e81d16993e1.png">

Reply:
It is okay to have on single enum for all the platforms and this enum can include Unknown. 
you already have static SupportedWallets - use it to handle different platforms.